src/main/java/com/statsig/androidlocalevalsdk/Evaluator.kt

```diff
@@ -210,6 +210,10 @@ internal class Evaluator(private val specStore: Store, private val errorBoundary
                     value = EvaluatorUtils.getFromUser(user, field)
                 }
 
+                ConfigCondition.UA_BASED -> {
```

Comment:
Can't you stack cases like in JS?
```suggestion
                ConfigCondition.IP_BASED ->,
                ConfigCondition.UA_BASED ->,
```

Reply:
yeah probably